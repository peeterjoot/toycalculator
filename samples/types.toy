BOOL i1;
i1 = TRUE;
PRINT i1;

INT8 i8;
i8 = 10;
PRINT i8;

INT16 i16;
i16 = 1000;
PRINT i16;

INT32 i32;
i32 = 100000;
PRINT i32;

INT64 i64;
i64 = 100000000000;
PRINT i64;

FLOAT32 f32;
f32 = 1.1;
PRINT f32;

FLOAT64 f64;
f64 = 2.2E-1;
PRINT f64;
