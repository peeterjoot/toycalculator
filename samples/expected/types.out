1
10
1000
100000
100000000000
1.100000
0.220000
